sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -89,12 +95,101 @@ def impl(self):
     return impl
 
 
+def gen_hpat_pandas_series_rolling_zerominp_impl(rolling_func, output_type=None):
+    """Generate series rolling methods implementations with zero min_periods"""
+    nan_out_type = output_type is None
+
+    def impl(self):
+        win = self._window
+
+        input_series = self._data
+        input_arr = input_series._data
+        length = len(input_arr)
+        out_type = input_arr.dtype if nan_out_type == True else output_type  # noqa
+        output_arr = numpy.empty(length, dtype=out_type)
+
+        for i in prange(min(win, length)):
```

Comment:
A comment here regarding the size of the part of the array in window would be useful. Also we might call min(win, length) somehow.

Reply:
@kozlov-alexey not sure that you exactly mean by
>Also we might call min(win, length) somehow.

?

And by
>A comment here regarding the size of the part of the array in window would be useful.

too...

Reply:
@AlexanderKalistratov  I just suggested to use a new variable storing min(win, length), since it's used twice, but it's already done as I see.